devservices/config.yml

```diff
@@ -11,7 +11,10 @@ services:
   redis:
     image: ghcr.io/getsentry/image-mirror-library-redis:5.0-alpine
     healthcheck:
-      test: redis-cli ping
+      test: redis-cli ping | grep PONG
```

Comment:
Why? Since we should wait until redis has loaded the dataset

https://stackoverflow.com/questions/67904609/how-do-you-perform-a-healthcheck-in-the-redis-docker-image